test/plugins/inlineStyles.15.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Yes, this is intentional, it skips selectors that cause problems instead failing.
I could add an option to the plugin for not showing these warnings.